profiling/summary.md

```diff
@@ -0,0 +1,3 @@
+Summary for profiling.
+During investigation profile my application I didn't find any big troubles with CPU/Memory. 
+For memory optimization I can use in application library [fastjson](https://github.com/valyala/fastjson) for validation and parsing JSON data instead of `json.Decode` and `go-playground/validator`. 
```

Comment:
Describe cases where you used concurrency and why.
What about logging? Does current logger is able to write logs asynchronously? Which libs you can also use?
Describe why you used standart go router, if there is another router with better performance.